internal/generate/generate_helper.go

```diff
@@ -45,6 +45,19 @@ func GenerateArtifactsHelper(t *templator.Templator, cfg *config.Commit0Config,
 		kubernetes.Generate(t, cfg, &wg, pathPrefix)
 	}
 
+	log.Println(aurora.Cyan(emoji.Sprintf("Generating Terraform")))
+	terraform.Generate(t, cfg, &wg, pathPrefix)
+	if cfg.Infrastructure.AWS.Cognito.Deploy {
+
+		outputs := []string{
+			"cognito_pool_id",
+			"cognito_client_id",
+		}
+		outputValues := terraform.ExecuteWithOuput(cfg, pathPrefix, outputs)
+		cfg.Frontend.Env.CognitoPoolID = outputValues["cognito_pool_id"]
+		cfg.Frontend.Env.CognitoClientID = outputValues["cognito_client_id"]
```

Comment:
I wonder if we should write these values back to the `commit0.yml` file..

Reply:
yup for sure, I think we definitely should do that